bot.py

```diff
@@ -115,6 +115,10 @@ async def help(self, ctx: commands.Context):
     async def info(self, ctx: commands.Context):
         await ctx.reply(embed=constants.INFO_REPLY)
 
+    @commands.hybrid_command()
+    async def error(self, ctx:commands.Context):
+        raise Exception("peepee")
+
```

Comment:
I'm sorry, I'm going to need an explanation on this one.

Reply:
THAT WAS TESTING CODE I DID **NOT** MEAN TO COMMIT THAT OH MY GOD